packages/ui-components/src/components/DataTable/StyledTable.tsx

```diff
@@ -5,58 +5,22 @@
 import styled from 'styled-components';
 import MuiTable from '@material-ui/core/Table';
 
-const DARK_THEME_BORDER = 'rgb(82, 82, 88)';
-
 export const StyledTable = styled(MuiTable)`
   table-layout: fixed;
-  overflow: hidden;
-  border: 1px solid ${({ theme }) => (theme.palette.type === 'light' ? 'none' : DARK_THEME_BORDER)};
 
```

Comment:
These styles were moved out of `ui-components` and into the theme of the package that uses it, so it's easy to change these wherever we want to